README.md

```diff
@@ -31,6 +31,10 @@ npm start
 
 Learn more about Electron and its API in the [documentation](http://electron.atom.io/docs/).
 
+---
+Warning: Windows 10 users might encounter some errors while running `npm start` from Linux Bash for Windows. The reason for this is that Electron is a GUI application, but Linux bash for Windows is just a headless VM without any GUI ( [explained here](https://github.com/electron/electron-quick-start/issues/129#issuecomment-279701745) ).
+A temporary workaround is getting [NodeJS for Windows](https://nodejs.org/en/) and use the node cmd.
```

Comment:
As a kinda Windowsy dev I can say a few things about this

* Doing Electron development in the Windows Linux subsystem **is not** recommended as you are literally developing a Windows application from a Linux operating system
* Using `node` and `npm` from Command Prompt is (at least IMO) the recommended way to use Node.JS in Windows
* This isn't a temporary issue with a temporary workaround, this is a fundamental issue with what you are trying to achieve.  Running GUI applications from Ubuntu for Windows is simply not designed to work 👍 

On a side note, I've had mild success with [this guide](https://www.howtogeek.com/261575/how-to-run-graphical-linux-desktop-applications-from-windows-10s-bash-shell) in the past when I was bored for a few hours 😆 but I have never tested it with Electron.